doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,368 @@
+
+Installing the |NCS| 2.6.99-cs2
+*******************************
+
+After you have installed the toolchain, complete the following steps to get the |NCS| v2.6.99-cs2:
+
+1. In the terminal window opened during the toolchain update, enter the following command to clone the project repositories::
+
+      west update
+
+   Depending on your connection, this might take some time.
+
+#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
+   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
+
+      west zephyr-export
+
+Your directory structure now looks similar to this::
+
+    ncs-lcs/work-dir
+    |___ .west
+    |___ bootloader
+    |___ modules
+    |___ nrf
+    |___ nrfxlib
+    |___ zephyr
+    |___ ...
+
+Note that there are additional folders, and that the structure might change.
+The full set of repositories and folders is defined in the manifest file.
+
+Updating the Terminal application
+*********************************
+
+To update `nRF Connect Serial Terminal`_, follow these steps:
+
+1. On your computer, open `nRF Connect for Desktop`_
+   If there is an update available, a pop up will notify you of its availability.
+#. If available, install the update from the pop up screen.
+#. Update `nRF Connect Serial Terminal`_ from the nRF Connect for Desktop application)
+
+If you are using the nRF Terminal application part of the `nRF Connect for Visual Studio Code`_ extension, open Visual Studio Code instead and ensure you are running the newest version of both the editor and the extension.
+
+.. _migration_cs3_to_2_6_99_cs2_env_bringup:
+
+nRF54H20 bring-up
+*****************
+
+The following sections describe the steps required for the nRF54H20 bring up.
+
+.. _migration_cs3_to_2_6_99_cs2_env_fwbundle:
+
+Installing the nRF54H20 firmware bundle
+=======================================
+
+The nRF54H20 SoC requires the installation of a bundle ( :file:`nrf54h20_soc_binaries_v0.3.0.zip`) containing the precompiled firmware for the Secure Domain and System Controller:
+
+..
+   ### Update bundle link
+
+1. Download the `nRF54H20 firmware bundle`_.
+#. Move the :file:`ZIP` bundle to a folder of your choice, then indicate the bundle folder path to nRF Util using the following command::
+
+      nrfutil device x-provision-nrf54h --firmware *path-to_bundle_.zip_file* --serial-number *serialnumber*
+
+Programming the BICR
+====================
+
+To prepare the nRF54H20 SoC for first use, you must also manually program the values of the BICR registers using the BICR binary file.
+
+1. Download the ``file`` here.
+#. Move the BICR file to a folder of your choice, then program the BICR by running nRF Util from that folder using the following command::
+
+   nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware bicr_ext_loadcap.hex --core Secure --serial-number $snr
+
+..
+   update file link
+
+Programming the SDFW and SCFW from the bundle
```

Comment:
Isn't it the same steps as "Installing the nRF54H20 firmware bundle" ?

Reply:
No, these program the firmware, the chapter you mentioned points nrfutil to where to find the files